### ISSUE TITLE
Server won't start (fails with JavaScript error)

### ISSUE THREAD

**Issue by tesbreag** [fttx/barcode-to-pc-server#38]
New to GitHub, so sorry if this is inappropriate, but struggling to find any other form of support forum for this program!

I installed on Windows 10, and the first run attempt resulted in an instant JavaScript error:

```
Uncaught Exception
Error: DNSServiceRef is not initialized
   at....(loads of different locations)
```

Any idea what's going wrong here?  This program would be very useful to me, so I hope so!

Thanks!

![errorbtp](https://cloud.githubusercontent.com/assets/27592628/25067751/205fc880-2245-11e7-804e-021b11393e40.PNG)

**Comment by fttx** [comment]
It was my fault, you need to install the Apple Bonjour SDK, i forgot to include it in the installer.

Or download the previous version of the server: https://github.com/fttx/barcode-to-pc-server/releases/download/v1.0.0/Barcode.to.PC.server.Setup.1.0.0.exe

**Comment by tesbreag** [comment]
Hello,
Thanks for the reply.

Sadly, there only seems to be a 64-bit version of that earlier release, and the Apple website won't let me register to download Bonjour from there (my hatred of all things Apple just went up a notch after filling out the registration form a dozen times, only for it not to accept the verification code sent to my email! every time with an "unknown error"... just useless).

So unless there's a third-party site where I can download that SDK, I guess I have no option but to wait patiently for another release that includes the Bonjour installer.  Any idea when that might be?

Thanks again.

**Comment by fttx** [comment]
I don't know how long it will take because to include it in the installer i must mail (post) the license agreements to Apple first.

I think that for now i'll stick with the old library https://github.com/watson/bonjour that unfortunatelly has this issue: https://github.com/watson/bonjour/issues/30

**Comment by tesbreag** [comment]
I see.  How frustrating.  Is there a 32-bit version of the old release?

**Comment by fttx** [comment]
https://github.com/fttx/barcode-to-pc-server/releases/download/v1.1.0-rc2/Barcode.to.PC.server.Setup.1.1.0.x32.exe

**Comment by fttx** [comment]
Fixed in v3.0.2
